_layouts/docs.html

```diff
@@ -177,11 +177,16 @@ <h2>Next steps</h2>
 							{{ content }}
 							<!-- tags -->
 							{% unless page.notags == true %}
-							{% assign keywords = page.keywords | split:"," %}
-							{% for keyword in keywords %}{% assign strippedKeyword = keyword | strip %}
-							{% capture keywordlist %}{{ keywordlist }}<a href="https://docs.docker.com/search/?q={{strippedKeyword}}">{{strippedKeyword}}</a>{% unless forloop.last %}, {% endunless %}{% endcapture %}
-							{% endfor %}
-							{% if keywordlist.size > 0 %}<span class="glyphicon glyphicon-tags" style="padding-right: 10px"></span><span style="vertical-align: 2px">{{ keywordlist }}</span>{% endif %}
+							{% assign keywords = page.keywords | split:"," -%}
+							{% if keywords.size > 0 -%}
+							  <span class="glyphicon glyphicon-tags" style="padding-right: 10px"></span><span style="vertical-align: 2px">
```

Comment:
Just noticing this here (not needed to change as part of this PR); we should get rid of these inline styles, and either add a CSS class or add a style for `glyphicon` /  `glyphicon-tags` to our stylesheet to do the padding.

Also need to check why we're adding the extra `<span>` here, and if instead we could use the `<a>` itself to do this (haven't tried yet)